src/bookshelf/rebar.config

```diff
@@ -31,7 +31,7 @@
 
 {erl_opts, [debug_info,
             {parse_transform, lager_transform},
-            warnings_as_errors}].
+            warnings_as_errors]}.
```

Comment:
My guess was that it wasn't but we didn't notice because rebar.config.lock gets used instead for nearly all builds.